src/zoo.js

```diff
@@ -49,18 +93,54 @@ function animalMap(options) {
 
 function schedule(dayName) {
   // seu código aqui
+  // const schedule = (dayName) => {
+  // const objectHours = Object.entries(data.hours);
+  // console.log (objectHours);
+  // };
 }
 
 function oldestFromFirstSpecies(id) {
   // seu código aqui
+  const oldAnimal = Object.values(data.animals.find(el => el.id === data.employees
+    .find(animal => animal.id === id).responsibleFor[0]).residents
+    .sort((first, second) => second.age - first.age)[0]);
+  return oldAnimal;
 }
 
 function increasePrices(percentage) {
   // seu código aqui
+  Object.keys(data.prices).forEach((value) => {
+    (data.prices[value] = Math.round(data.prices[value] * (1 + (percentage / 100)) * 100) / 100);
+  });
 }
 
 function employeeCoverage(idOrName) {
   // seu código aqui
+  const list = {};
+  const employeeFunction = (employee) => {
+    list[`${employee.firstName} ${employee.lastName}`] = employee
+      .responsibleFor.map(functionJob => data.animals
+        .find(animal => animal.id === functionJob).name);
+  };
+  const employeeResponsibility = () => {
+    const responsibility = data.employees
+      .filter(employ => employ.id === idOrName
+        || employ.firstName === idOrName || employ.lastName === idOrName);
+    list[`${responsibility[0].firstName} ${responsibility[0].lastName}`] = [];
+    employeeFunction(responsibility[0]);
```

Comment:
um find é melhor ja que vc so precisa encontrar 1, e da pra excluir a linha 129 já que a linha 121 faz exatamente o mesmo
```suggestion
      .find(employ => employ.id === idOrName
        || employ.firstName === idOrName || employ.lastName === idOrName);
    employeeFunction(responsibility);
```